pkg/typesystem/typesystem_test.go

```diff
@@ -495,40 +461,18 @@ func TestInvalidRewriteValidations(t *testing.T) {
 			err: ErrReservedKeywords,
 		},
 		{
-			name: "Fails_If_Auth_Model_1.0_Has_A_Cycle_And_Only_One_Type",
+			name: "Fails_If_Auth_Model_1.1_Has_A_Cycle_And_Only_One_Type",
 			model: &openfgapb.AuthorizationModel{
-				SchemaVersion: SchemaVersion1_0,
-				TypeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type: "folder",
-						Relations: map[string]*openfgapb.Userset{
-							"parent": This(),
-							"viewer": TupleToUserset("parent", "viewer"),
-						},
-					},
-				},
+				SchemaVersion: SchemaVersion1_1,
+				TypeDefinitions: parser.MustParse(`
+				type folder
+				  relations
+				    define parent: [folder] as self
+					define viewer as viewer from parent
+				`),
 			},
 			err: ErrCycle,
 		},
-		{
-			name: "Succeeds_If_Auth_Model_1.0_Has_A_Cycle_But_There_Is_More_Than_One_Type",
```

Comment:
This test and the removed above are no longer relevant/possible under a 1.1 model paradigm.